Dockerfile

```
@@ -23,7 +23,6 @@ COPY . .
 RUN cargo build --release
 
 FROM base AS tester
-COPY --from=builder /app/target/release/fennel-node /app/fennel-node
 RUN cargo test --features=runtime-benchmarks
```

I screwed this up again - your suggestion was closer.
```suggestion
FROM base AS tester
COPY . .
RUN cargo test --features=runtime-benchmarks
```
This'll copy the source code to the tester layer and run all our tests in Docker.

---

np. i will update again. ty